### PR TITLE
[WIP] Began adding tavern and party

### PR DESCRIPTION
**Pull request by TheHollidayInn**

### CODE REVIEW

File: website/client/components/appMenu.vue [hunk 1]
```diff
@@ -14,7 +14,9 @@ nav.navbar.navbar-inverse.fixed-top.navbar-toggleable-sm
           router-link.dropdown-item(:to="{name: 'stable'}") {{ $t('stable') }}
       router-link.nav-item(tag="li", :to="{name: 'shops'}", exact)
         a.nav-link(v-once) {{ $t('shops') }}
-      router-link.nav-item(tag="li", :to="{name: 'party'}", exact)
+      router-link.nav-item(tag="li", :to="{name: 'market'}", exact)
```
Comment (paglias) [inline website/client/components/appMenu.vue]: I think market is now under shops, so this link can be removed

File: website/client/components/appMenu.vue [hunk 2]
```diff
@@ -14,7 +14,9 @@ nav.navbar.navbar-inverse.fixed-top.navbar-toggleable-sm
...
+        a.nav-link(v-once) {{ $t('market') }}
+      router-link.nav-item.dropdown(:to="{name: 'party'}", :class="{'active': $route.path.startsWith('/party')}")
```
Comment (paglias) [inline website/client/components/appMenu.vue]: is the :class binding necessary? It should work without it

File: website/client/components/guilds/guild.vue [hunk 3]
```diff
@@ -267,7 +267,13 @@ export default {
     },
   },
   created () {
+    if (this.$route.path.startsWith('/party')) {
+      this.guildId = 'party';
+    } else if (this.$route.path.startsWith('/guilds/tavern')) {
+      this.guildId = 'habitrpg';
```
Comment (paglias) [inline website/client/components/guilds/guild.vue]: I think this is not anymore `habitrpg` but common/constants/TAVERN_ID

File: website/client/router.js [hunk 4]
```diff
@@ -55,11 +55,13 @@ export default new VueRouter({
       ],
     },
     { name: 'shops', path: '/shops', component: Page },
+    { name: 'market', path: '/market', component: Page },
```
Comment (paglias) [inline website/client/router.js]: shops

File: website/client/router.js [hunk 5]
```diff
@@ -55,11 +55,13 @@ export default new VueRouter({
...
+    { name: 'party', path: '/party', component: GuildPage },
```
Comment (paglias) [inline website/client/router.js]: This is a duplicate, there's already a party entry in the router

File: website/client/router.js [hunk 6]
```diff
@@ -101,6 +103,14 @@ export default new VueRouter({
             },
           ],
         },
+        { name: 'challenges', path: 'challenges', component: Page },
```
Comment (paglias) [inline website/client/router.js]: why challenges under user? also it's a duplicate, there's another challenge entry a few lines above

File: website/client/router.js [hunk 7]
```diff
@@ -101,6 +103,14 @@ export default new VueRouter({
...
+      ],
+    },
+    {
+      path: '/user',
```
Comment (paglias) [inline website/client/router.js]: this too seems a duplicate, maybe some type of conflict with `develop` but there's already an user entry

**Review by paglias — CHANGES_REQUESTED** [review]

File: website/client/router.js (hunk 7 above)
Comment (TheHollidayInn) [reply website/client/router.js]: Yea, I think I just messed up when merging the routes. I'll take a look at develop and fix it.

File: website/client/router.js [hunk 8]
```diff
@@ -81,7 +80,6 @@ export default new VueRouter({
       ],
     },
     { name: 'challenges', path: 'challenges', component: Page },
-    { name: 'party', path: 'party', component: Page },
     {
       path: '/user',
       component: ParentPage,
```
Comment (paglias) [inline website/client/router.js:86]: outside the scope of this PR but this component should be changed with one that contains the submenu, will do when we implement the users views

File: website/common/locales/en/newClient.json [hunk 9]
```diff
@@ -86,5 +86,10 @@
   "startYourOwnPartyDescription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam accumsan sagittis tellus tempor euismod. Sed imperdiet facilisis tortor in malesuada.",
   "shartUserId": "Share User ID",
   "wantToJoinPartyTitle": "Want to join a Party?",
-  "wantToJoinPartyDescription": "Aenean non mattis eros, quis semper ipsum. Phasellus vulputate in nibh et suscipit. In hac habitasse platea dictumst."
+  "wantToJoinPartyDescription": "Aenean non mattis eros, quis semper ipsum. Phasellus vulputate in nibh et suscipit. In hac habitasse platea dictumst.",
+  "copy": "Copy",
+  "lookingForGroup": "Looking for Group",
+  "qrCode": "QR code",
+  "facebook": "Facebook",
```
Comment (paglias) [inline website/common/locales/en/newClient.json]: I don't think we should translate Facebook and Twitter

Comment (TheHollidayInn) [reply website/common/locales/en/newClient.json]: Ah, that makes sense haha

File: website/client/assets/svg/quest-background-border.svg [hunk 10]
```diff
@@ -6,7 +6,7 @@
         <path id="d" d="M0 70h10v10H0z"/>
         <path id="e" d="M414 70h10v10h-10z"/>
     </defs>
-    <g fill="none" fill-rule="evenodd">
+    <g fill="#F9F9F9" fill-rule="evenodd">
```
Comment (paglias) [inline website/client/assets/svg/quest-background-border.svg]: just fyi, if you remove the fill attribute then you can style the icon with a css `color: XXXX`

File: website/client/components/guilds/startQuestModal.vue [hunk 11]
```diff
@@ -89,9 +132,26 @@ export default {
...
+      return quests.quests[this.selectedQuest];
+    },
+  },
+  methods: {
+    async questInit () {
+      let key = this.selectedQuest;
+      // Analytics.updateUser({'partyID': party._id, 'partySize': party.memberCount});
+      let response = await this.$store.dispatch('guilds:inviteToQuest', {groupId: this.group._id, key});
+      let quest = response.data.data;
+      this.group.quest = quest;
```
Comment (paglias) [inline website/client/components/guilds/startQuestModal.vue]: why both this.group and this.$store.party?

Comment (TheHollidayInn) [reply website/client/components/guilds/startQuestModal.vue]: That is just from experimenting. I had local only, but then needed to share.

File: website/client/components/guilds/createPartyModal.vue [hunk 12]
```diff
@@ -0,0 +1,121 @@
...
+  }
+
+  .share-userid-options {
+    background-color: $white;
+    border-radius: 2px;
+    width: 180px;
+    position: absolute;
+    top: -8em;
+    left: 4.8em;
+    box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.16), 0 1px 4px 0 rgba(26, 24, 29, 0.12);
```
Comment (paglias) [inline website/client/components/guilds/createPartyModal.vue:60]: $black

File: website/client/components/guilds/groupFormModal.vue [hunk 13]
```diff
@@ -70,13 +74,26 @@
               span.custom-control-description(v-once) {{ $t(group.label) }}
           button.btn.btn-primary(@click.prevent="toggleCategorySelect") {{$t('close')}}
 
+      .form-group(v-if='inviteMembers && !newGuild.id')
+        label
+          strong(v-once) Invite via Email or User ID
```
Comment (paglias) [inline website/client/components/guilds/groupFormModal.vue:79]: i18n

File: website/client/components/guilds/membersModal.vue [hunk 14]
```diff
@@ -45,7 +49,7 @@ div
     button(@click='confirmRemoveMember(member)', v-once) {{$t('remove')}}
 </template>
 
-<style lang='scss'>
+<style lang='scss' scoped>
   header {
     background-color: #edecee;
```
Comment (paglias) [inline website/client/components/guilds/membersModal.vue:54]: color var

File: website/client/components/guilds/membersModal.vue [hunk 15]
```diff
@@ -66,6 +70,49 @@ div
...
+      margin: 0;
+    }
+
+    .member-stats {
+      width: 382px;
+      height: 147px;
+    }
+
+    .gradient {
+      background-image: linear-gradient(to bottom, rgba(255, 255, 255, 0), #ffffff);
```
Comment (paglias) [inline website/client/components/guilds/membersModal.vue:91]: color vars

File: website/client/components/guilds/membersModal.vue [hunk 16]
```diff
@@ -66,6 +70,49 @@ div
...
+      height: 200px;
+      width: 100%;
+      position: absolute;
+      bottom: 0px;
+    }
+  }
+
+  .item-with-icon {
+    border-radius: 2px;
+    background-color: #ffffff;
```
Comment (paglias) [inline website/client/components/guilds/membersModal.vue:101]: color vars

**Review by paglias — COMMENTED** [review]
there are a lot of missing i18n strings / color vars + can we add TODOs for all the code that is commented but not implemented, otherwise I have no idea of what's done and what's missing

**Pull request merged**